tests/unit/game_conrtoller_test.py

```diff
@@ -0,0 +1,86 @@
+from unittest import TestCase
+
+from game_controller import GameController
+
+
+class GameControllerTest(TestCase):
+    def setUp(self):
```

Comment:
Tests are dependent from each other. Not sure how to fix this.

Reply:
Tests aren't dependent on each other and can be run by itself.